datastore/cloud-ndb/named_db_test.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+import backoff
+import pytest
+
+import named_db
+
+
+@pytest.fixture
+def test_book():
+    book = named_db.Book(title=str(uuid.uuid4()))
+    # The setup and teardown (put and delete) are done in separate contexts
+    # only to ensure that the test phase in the middle handles contexts on its
+    # own correctly. It is normally desirable to have all related sequential
+    # ndb calls in the same context.
+    with named_db.client.context():
+        book.put()
+    yield book
+    with named_db.client.context():
+        book.key.delete()
+
+
+def test_named_db(capsys, test_book):
+    @backoff.on_exception(backoff.expo, AssertionError, max_time=60)
```

Comment:
suggestion: consider removing the `eventually_consistest_test()` function and moving the backoff annotation to line 36 to cover whole `test_named_db`.